scripts/services/getConfig.ts

```diff
@@ -178,6 +178,13 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
         "name": "Mempool",
         "description": "Mempool Settings",
         "spec": {
+          "mempoolfullrbf": {
+            "name": "Enable Full RBF",
+            "description":
+              "Policy for your node to use for relaying and mining unconfirmed transactions.  For details, see https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-24.0.md",
```

Comment:
Link to the actual section where it mentions full rbf?

```suggestion
              "Policy for your node to use for relaying and mining unconfirmed transactions.  For details, see https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-24.0.md#notice-of-new-option-for-transaction-replacement-policies",
```